docs/usage/channels.rst

```diff
@@ -226,81 +239,70 @@ subscriptions need to be managed dynamically.
 Or, using the context manager
 
 .. code-block:: python
+    :caption: Using the :term:`async context manager <asynchronous context manager>` to unsubscribe from a
+      :term:`channel`
 
     async with channels.start_subscription(["foo", "bar"]) as subscriber:
         ...  # do some stuff here
         await channels.unsubscribe(subscriber, ["foo"])
 
+Managing :term:`history`
+++++++++++++++++++++++++
 
-
-Managing history
-+++++++++++++++++
-
-Some backends support per-channel history, keeping a certain amount of
-:term:`events <event>` in storage. This history can then be pushed to a
-:term:`subscriber`.
+Some backends support per-:term:`channel` :term:`history`, keeping a certain amount of :term:`events <event>` in storage.
+This :term:`history` can then be pushed to a :term:`subscriber`.
 
 The plugin's :meth:`put_subscriber_history <ChannelsPlugin.put_subscriber_history>` can
-be used to fetch this history and put it into a subscriber's :term:`event stream`.
+be used to fetch this :term:`history` and put it into a subscriber's :term:`event stream`.
 
 .. literalinclude:: /examples/channels/put_history.py
-    :language: python
-
+    :caption: Retrieving :term:`channel` :term:`history` for a :term:`subscriber` and putting it into the
+      :term:`stream <event stream>`
 
-.. note::
-    The publication of the history happens sequentially, one channel and one
-    event at a time. This is done to ensure the correct ordering of events and to avoid
-    filling up a subscriber's backlog, which would result in dropped history entries. Should the
+.. note:: The publication of the :term:`history` happens sequentially, one :term:`channel` and one
```

Comment:
```suggestion
.. note:: 
    The publication of the :term:`history` happens sequentially, one :term:`channel` and one
```
![Screenshot from 2024-04-09 18-16-24](https://github.com/litestar-org/litestar/assets/25355197/4211a27e-d692-4716-86ad-ac3a1fab62db)

Reply:
This seems to be a spacing issue it shouldnt do this, everywhere else i've done this it is proper. Will fix